config/routes.rb

```diff
@@ -11,14 +11,24 @@
       delete 'sign_out', :to => 'devise/sessions#destroy', :as => :destroy_auth_user_session
     end
 
+  resources :skills do
+    collection do
+      get 'export', to: 'skills#export'
+    end
+  end
+
+  resources :people_skills do
+    collection do
+      get 'filter_form', to: "people_skills/filter_form#index"
```

Comment:
can we restrict this to turbo only?

Reply:
We are going to restrict the turbo-controllers in another ticket as discussed.